src/browser/modules/DBMSInfo/MetaItems.jsx

```diff
@@ -56,14 +56,57 @@ const ShowMore = ({ total, shown, moreStep, onMore }) => {
   )
 }
 
+function createStyleGetter(graphStyleData, kind) {
+  const graphStyle = neoGraphStyle()
+  if (graphStyleData) {
+    graphStyle.loadRules(deepmerge(graphStyle.toSheet(), graphStyleData || {}))
+  }
+
+  if (kind === 'node') {
+    return function(text) {
+      if (graphStyleData) {
+        const styleForItem = graphStyle.forNode({
+          labels: [text]
+        })
+        return {
+          backgroundColor: styleForItem.get('color'),
+          color: styleForItem.get('text-color-internal')
+        }
+      }
+      return {}
+    }
+  }
+  if (kind === 'relationship') {
+    return function(text) {
+      if (graphStyleData) {
+        const styleForItem = graphStyle.forRelationship({
+          type: text
+        })
+        return {
+          backgroundColor: styleForItem.get('color'),
+          color: styleForItem.get('text-color-internal')
+        }
+      }
+      return {}
+    }
+  }
+  throw new Error(`Unsupported argument: ${kind}`)
+}
+function createNodeStyleGetter(graphStyleData) {
+  return createStyleGetter(graphStyleData, 'node')
+}
+function createRelationStyleGetter(graphStyleData) {
```

Comment:
should be named `createRelationshipStyleGetter` to use neo4j terminology

Reply:
Done!